tests/positive/partitions/creation-mb.go

```diff
@@ -26,6 +26,7 @@ func init() {
 	register.Register(register.PositiveTest, WipeAndCreateNewPartitionsMiB())
 	register.Register(register.PositiveTest, AppendPartitionsMiB())
 	register.Register(register.PositiveTest, ResizeRootMiB())
+	register.Register(register.PositiveTest, ResizeExistingPartitionsMiB())
```

Comment:
It wouldn't hurt to also have a negative test in which we try to resize a partition which doesn't match in one of the fields, such as the starting offset.